packages/react/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import {Button, Box} from '..'
+import {Tooltip} from './'
+
+/* Tooltip v1 */
+
+export default {
+  title: 'Components/Tooltip',
+  component: Tooltip,
+} as Meta<typeof Tooltip>
+
+export const Default = () => (
+  <Box padding="spacious">
+    <Tooltip text="Hello, Tooltip!">
+      <Button>Hover me</Button>
+    </Tooltip>
+  </Box>
+)
+
+export const Playground = args => {
+  // this is a hack to remove the `type` prop from the args because for this example type label is not a valid choice and violates accessibility
```

Comment:
👍 this gets picked up in Storybook controls too...

<img width="630" alt="Screenshot 2024-03-26 at 15 08 37" src="https://github.com/primer/brand/assets/13340707/9e19debe-8140-4fe5-901e-1d5555e60349">

Should be able to remove it with:

```js
type: {
  table: {
    disable: true,
  },
},
```